code/modules/antagonists/heretic/knowledge/sacrifice_knowledge/sacrifice_knowledge.dm

```diff
@@ -374,8 +373,11 @@
 	sac_target.remove_status_effect(/datum/status_effect/unholy_determination)
 	sac_target.reagents?.del_reagent(/datum/reagent/inverse/helgrasp/heretic)
 	sac_target.clear_mood_event("shadow_realm")
-	sac_target.gain_trauma(/datum/brain_trauma/mild/phobia/supernatural, TRAUMA_RESILIENCE_MAGIC)
-
+	if(IS_HERETIC(sac_target))
+		var/datum/antagonist/heretic/victim_heretic = sac_target.mind?.has_antag_datum(/datum/antagonist/heretic)
+		victim_heretic.knowledge_points -= 3
```

Comment:
i feel like i SHOULD make this into some kind of var but i don't know where i'd place this var